schema/instrumentation.json

```diff
@@ -88,6 +88,12 @@
             "type": "object",
             "additionalProperties": false,
             "properties": {
+                "known_methods": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
```

Comment:
maybe this should be split into client and server options

I could almost see this option not being needed on client spans since it is primarily needed for avoiding metric cardinality attacks on servers (but I see we've defined it on both client and server spans in semconv, I don't remember if we discussed this previously, @lmolkova?)